infra/modules/providers/azure/app-service/main.tf

```diff
@@ -91,6 +92,12 @@ resource "azurerm_app_service_slot" "appsvc_staging_slot" {
   depends_on          = [azurerm_app_service.appsvc]
 }
 
+data "azurerm_app_service" "all" {
```

Comment:
Is this required? I see that the azurerm_template_deployment.access_restriction depends on this, but it doesn't look as though the data is actually used otherwise.

Reply:
I think I gave the impression that this wasn't needed but it's actually used in the outputs.tf of the iso template.